IDE/VisualDSP/include.am

```diff
@@ -0,0 +1,6 @@
+# vim:ft=automake
+# included from Top Level Makefile.am
+# All paths should be given relative to the root
+
+EXTRA_DIST+= IDE/VisualDSP++/user_settings.h
```

Comment:
These look wrong... `VisualDSP++`. First I think its `IDE/VisualDSP` and also we should avoid using '+' in file or directory names.